addons/hr/views/res_users.xml

```diff
@@ -257,13 +257,12 @@
                             <!-- share is not correctly recomputed because it depends on fields of reified view => invisible before saving (id=False) -->
                 </xpath>
                 <xpath expr="//div[@name='button_box']" position="inside">
-                    <button name="%(hr_employee_action_from_user)d"
+                    <button name="action_open_employees"
                         class="oe_stat_button"
                         icon="fa-users"
                         attrs="{'invisible': [('employee_count', '=', 0)]}"
-                        context="{'active_test': False}"
```

Comment:
Why did we remove this context? :thinking: What it will do is the following  - if we have an archived employee linked with the user, once we redirect, we will see it. Now, after removing this, it won't be the case.